lib/world_cup.rb

```diff
@@ -0,0 +1,30 @@
+class WorldCup
+  attr_reader :year,
+              :teams
+
+  def initialize(year, teams)
+    @year = year
+    @teams = teams
+  end
+
+  def teams_not_eliminated
+    @teams.find_all do |team|
+      team.eliminated? == false
+    end
+  end
+
+  def active_players_by_position(player_position)
+    active_players = []
+    teams_not_eliminated.each do |team|
+      team.players.find_all do |player|
+        if player.position == player_position
+          active_players << player
+        end
```

Comment:
This code looks very similar to the code in the Team class to me around finding `players_by_position`. Could potentially refactor to something like the following:

```ruby
def active_players_by_position(player_position)
  teams_not_eliminated.map do |team|
    team.players_by_position(player_position)
  end.flatten
end
```